src/components/productDetails/card.jsx

```diff
@@ -54,7 +54,7 @@ const Card = ({
             <i className="fas fa-star"></i>
           )}
         </button>
-        <figure className="image is-4by3">
+        <figure className="image is-square">
```

Comment:
this solves some issues, but stretches the images. maybe use a container like I did in the styles component in overview?

Reply:
I will look into it if I have time this afternoon! Good idea.